pkg/api/ingest_kafka.go

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (C) 2022 IBM, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package api
+
+type IngestKafka struct {
+	Brokers          []string `yaml:"brokers" doc:"list of kafka broker addresses"`
+	Topic            string   `yaml:"topic" doc:"kafka topic to listen on"`
+	GroupId          string   `yaml:"groupid" doc:"separate groupid for each consumer on specified topic"`
+	GroupBalancers   []string `yaml:"groupbalancers" doc:"list of balancing stategies (range, roundRobin, rackAffinity)"`
```

Comment:
I am asking myself if we need to change the yaml format to `groupBalancers` (i.e. use capital letter for the second word .... this applies to all ... not sure what make more sense ?)